packages/boilerplate-generator/boilerplate_web_browser_template.js

```diff
@@ -0,0 +1,70 @@
+// Template function for rendering the boilerplate html for browsers
+// Replicates the template defined in boilerplate_web.browser.html
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+export default function (manifest) {
+  const root = manifest;
+  // XXX do we need to do some validation on the properties of root?
+  return [].concat(
+    [
+      '<html' +_.map(root.htmlAttributes, (value, key) =>
+        _.template(' <%= attrName %>="<%- attrValue %>"')({
+          attrName: key,
```

Comment:
I agree that it's a bit confusing, but I was hoping that it would make sense:
- `(value, key)` correspond to `{key: value}` pairs
- `attrName` and `attrValue` are are templated as `name="value"` HTML attributes.

I'm happy to change it to be ``<% key %.="<%- value %>"'` if you think that's less confusing.